docs/reference/installation.rst

```diff
@@ -5,34 +5,55 @@
 Installation
 ============
 
-The easiest way to install ``SonataTimelineBundle`` is to require it with Composer:
+Prerequisites
+-------------
 
-.. code-block:: bash
+PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand.
 
-    composer require sonata-project/timeline-bundle
+Required dependencies:
+
+* `SonataBlockBundle <https://sonata-project.org/bundles/block>`_
+
+And the persistence bundle:
+
+* `SonataDoctrineOrmAdminBundle <https://sonata-project.org/bundles/doctrine-orm-admin>`_
```

Comment:
I guess this is not the only available persistence bundle.

Reply:
It is for this concrete bundle. The other ones are not supported by this.

Reply:
Fixed here too.